readinglist/utils.py

```diff
@@ -18,9 +18,19 @@ def native_value(value):
         value = False
     try:
         return ast.literal_eval(value)
-    except ValueError:
+    except (ValueError, SyntaxError):
         return value
 
 
 def Enum(**enums):
     return type('Enum', (), enums)
+
+
+COMPARISON = Enum(
+    LT='<',
+    MIN='>=',
+    MAX='<=',
+    NOT='!=',
+    EQ='==',
+    GT='>',
```

Comment:
It is what I did and we finally decided it wouldn't make sense since it is related to the memory backend only.